alpine/APKBUILD.in

```diff
@@ -58,7 +58,6 @@ makedepends="
 	readline-dev
 	rpcgen
 	snappy-dev
-	userspace-rcu-dev
```

Comment:
@dx9 shall we drop this from this repo? as i think you've submitted the recipe to aport. see https://git.alpinelinux.org/aports/tree/community/ceph .

Reply:
@iggy Hi Iggy, i found you were updating Ceph's aport recently. could you help review this change?

Reply:
@tchaikov OK with me

Reply:
@dx9 thanks!

Reply:
will remove alpine packaging in a follow-up PR.